gatsby-config.js

```diff
@@ -6,7 +6,12 @@
 
 module.exports = {
   siteMetadata: {
-    title: `🍾 🤣 🍾`,
+    title: `🍾 🤣 🍾 Bottled Laughter`,
+    description: `A Bottle of Laughter by or Daughter`,
+    author: `@raae`,
+    keywords: `Bottled Happiness`,
```

Comment:
This should be a comma seperated list "happiness, silly site challenge, gatsby" etc. Not sure how much it has to say. It can also be blank.